lib/spell.js

```diff
@@ -142,13 +151,43 @@ function parseLogicalObjectConditionValue(value) {
  *     { content: 'Diablo' },
  *   ],
  * }
+ * {
+ *   title: {
+ *     $or: [
+ *       'Leah',
+ *       'Diablo',
+ *     ]
+ *   }
+ * }
+ * {
+ *   title: {
+ *     $or: [
+ *       'Leah',
+ *       {
+ *         $like: '%jjj'
+ *       },
+ *     ]
+ *   }
+ * }
+ * {
+ *   title: {
+ *     $not: [
+ *       'Leah',
+ *       'jss'
+ *     ]
```

Comment:
does this get translated into `title NOT IN ('Leah', 'jss')`?

Reply:
fixed